src/main/java/org/traccar/config/Keys.java

```diff
@@ -1162,6 +1162,27 @@ private Keys() {
             "notificator.telegram.sendLocation",
             List.of(KeyType.CONFIG));
 
+    /**
+     * Viber notification API key.
+     */
+    public static final ConfigKey<String> NOTIFICATOR_VIBER_KEY = new StringConfigKey(
+                    "notificator.viber.key",
+                    List.of(KeyType.CONFIG));
```

Comment:
Please fix indentation.

Reply:
What formatter are you using, because my Visual Studio code uses a different formatting and it changes formatting as soon as I save, I had to manually save without formatting.

Reply:
I use IntelliJ.